src/main.cc

```diff
@@ -1568,6 +1570,9 @@ SquidMain(int argc, char **argv)
 
         Mem::Init();
 
+        RunRegisteredHere(RegisteredRunner::bootstrapConfig);
+        // TODO: convert the below init() into RegisteredRunners::bootstrapConfig
```

Comment:
The comment seems misplaced (or oddly placed), and I doubt the proposed conversion is the best solution. I suggest dropping this out-of-scope change to avoid arguing about it.

Reply:
Done in c6b6dbd